### PR TITLE
Add 'ReadyTalk Desktop.app' as readytalk-desktop cask

### PR DESCRIPTION
**Pull request by dansomething**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Didn’t you get a written template when opening the PR?

File: Casks/readytalk-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'readytalk-desktop' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://core.callinfo.com/packages/runtime/rtd-readytalk/darwin-x86_64/setup.dmg'
```
Comment (vitorgalvao) [inline Casks/readytalk-desktop.rb]: Needs a `# callinfo.com was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by dansomething** [comment]
I apologize. I must have misread the PR template. I thought it said to remove the sections irrelevant to the change and then I ended up with nothing. Should I add back the full template?

**Comment by vitorgalvao** [comment]
There’s a section named “Editing an existing cask”. Why did that seem irrelevant to you? It’s the exact thing you’re doing here.

I’m really asking, as that template is quite new and we need to fix whatever is uncler.

**Comment by vitorgalvao** [comment]
There’s a section named [Adding a new cask](https://github.com/caskroom/homebrew-cask/blob/master/.github/PULL_REQUEST_TEMPLATE.md#adding-a-new-cask). Why did that seem irrelevant to you? It’s the exact thing you’re doing here.

I’m really asking, as that template is quite new and we need to fix whatever is unclear.

Yes, please add it back and do the checks.

**Comment by dansomething** [comment]
Sorry, I read  "- Delete every part of this placeholder text irrelevant to your submission, including this `Instructions` section." too literally. Does that mean to only apply to the first section? I believe after re-reading that you itended me to keep the "Adding" section since that is what I'm doing and then for me to put an "X" for each sub-item that I've completed?

**Comment by vitorgalvao** [comment]
Yes, that is correct. That first part is hopefully [clearer](https://github.com/caskroom/homebrew-cask/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions), now.

**Comment by dansomething** [comment]
It might help to follow the first sentence with something along the lines of "but be sure to keep the relevant section."

**Comment by vitorgalvao** [comment]
> It might help to follow the first sentence with something along the lines of "but be sure to keep the relevant section."

To be honest, I don’t see the sense in deleting all of template, *especially* since the line right under it tells you the one case where you can do so. I’d also like to keep it as small as possible.

If more people do it, we’ll revise it then. Right now, we need data.

**Comment by dansomething** [comment]
Fair enough. Thanks for the merge!

**Pull request merged**